odxtools/cli/dummy_sub_parser.py

```diff
@@ -1,6 +1,7 @@
 # SPDX-License-Identifier: MIT
 import argparse
 import sys
+from typing import List
```

Comment:
added but not used

Reply:
other than this, LGTM

Reply:
okay, I've removed all unused imports (at least those flagged by `ruff`): 58c013ac695db3e3687d87e83e2228d8769637cf